src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -158,13 +158,6 @@ public MediaRequestEntity updateAudioRequestStatus(Integer id, MediaRequestStatu
         return mediaRequestRepository.saveAndFlush(mediaRequestEntity);
     }
 
-    @Transactional
-    @Override
-    public MediaRequestEntity updateAudioRequestStatus(MediaRequestEntity mediaRequestEntity, MediaRequestStatus status) {
```

Comment:
author note:
this method is no longer used in production code so I'm removing it